### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by memoryleak47**

### CODE REVIEW

File: lang/types.md [hunk 1]
```diff
@@ -112,22 +112,22 @@ Here we define how to compute the size and other layout properties of a type.
...
-            Tuple { fields, .. } => fields.iter().all(|type| type.inhabited()),
-            Array { elem, count } => count == 0 || elem.inhabited(),
-            Union { .. } => true,
-            Enum { variants, .. } => variants.iter().any(|type| type.inhabited()),
+            Type::Int(..) | Type::Bool | Type::RawPtr { .. } => true,
+            Type::Ref { pointee, .. } | Type::Box { pointee } => pointee.inhabited,
+            Type::Tuple { fields, .. } => fields.iter().all(|type| type.inhabited()),
+            Type::Array { elem, count } => count == 0 || elem.inhabited(),
+            Type::Union { .. } => true,
+            Type::Enum { variants, .. } => variants.iter().any(|type| type.inhabited()),
```
Comment (RalfJung) [inline lang/types.md:130]: For these two I think a `use Type::*;` would be better.

File: lang/values.md [hunk 2]
```diff
@@ -313,17 +313,17 @@ For `Value`, we lift the order on byte lists to relate `Bytes`s, and otherwise r
...
+            (Value::Ptr(p1), Value::Ptr(p2)) =>
                 p1 <= p2,
-            (Tuple(vals1), Tuple(vals2)) =>
+            (Value::Tuple(vals1), Value::Tuple(vals2)) =>
                 vals1 <= vals2,
-            (Variant { idx: idx1, data: data1 }, Variant { idx: idx2, data: data2 }) =>
+            (Value::Variant { idx: idx1, data: data1 }, Value::Variant { idx: idx2, data: data2 }) =>
                 idx == idx1 && data1 <= data2,
-            (Union(chunks1), Union(chunks2)) => chunks1 <= chunks2,
+            (Value::Union(chunks1), Value::Union(chunks2)) => chunks1 <= chunks2,
```
Comment (RalfJung) [inline lang/values.md:326]: This should also have a `use`.

(Not sure where I put the threshold but when there are 12 uses in a single `match` it's definitely worth a `use`. Maybe the threshold is around 5.)

**Review by RalfJung — COMMENTED** [review]
Thanks! Just some style remarks.

**Comment by memoryleak47** [comment]
> Thanks! Just some style remarks.
Fixed!

**Comment by RalfJung** [comment]
There are conflicts with master, could you rebase and resolve them?

**Comment by memoryleak47** [comment]
> There are conflicts with master, could you rebase and resolve them?

Yep! didn't find the time yesterday.

**Comment by RalfJung** [comment]
Thanks!

**Pull request merged**